plugins/logs/plugin.go

```diff
@@ -829,6 +829,8 @@ func (p *Plugin) maskEvent(ctx context.Context, txn storage.Transaction, event *
 				rego.Store(p.manager.Store),
 				rego.Transaction(txn),
 				rego.Runtime(p.manager.Info),
+				rego.EnablePrintStatements(p.manager.EnablePrintStatements()),
+				rego.PrintHook(p.manager.PrintHook()),
 			)
 
 			pq, err := r.PrepareForEval(context.Background())
```

Comment:
I had _thought_ we'd need the EvalPrintHook() EvalOption in the call to Eval, _but_ I've convinced myself that we don't: the s.manager.PrintHook is the one of the runtime, which in turn calls the Logger. That thing can be called concurrently, so we're OK here.